features/plugins/git.feature

```diff
@@ -28,6 +28,7 @@ Feature: git Plugin
     * I successfully run `stove`
     * the git remote should have the tag "v0.0.0"
 
+  @wip
```

Comment:
I verified `master` and this scenario is also failing there, I'll create a card to fix it but for now,
I have marked as `@wip` so that we don't run it.

Reply:
Issue: https://github.com/chef/stove/issues/154